lib/ReactRotatingText.js

```diff
@@ -155,21 +170,23 @@ var ReactRotatingText = (function (_React$Component) {
   }, {
     key: 'render',
     value: function render() {
-      var _props2 = this.props;
-      var color = _props2.color;
-      var cursor = _props2.cursor;
-      var deletingInterval = _props2.deletingInterval;
-      var emptyPause = _props2.emptyPause;
-      var items = _props2.items;
-      var pause = _props2.pause;
-      var eraseMode = _props2.eraseMode;
-      var typingInterval = _props2.typingInterval;
-
-      var other = _objectWithoutProperties(_props2, ['color', 'cursor', 'deletingInterval', 'emptyPause', 'items', 'pause', 'eraseMode', 'typingInterval']);
+      var _props3 = this.props;
```

Comment:
Is this change necessary?

Reply:
@charj Same comment as above

Reply:
Therefore the lib needs rebuilding before this PR can be merged 👍

Reply:
@charj Sounds good.  I merged the latest changes and re-built.